saladItem.js

```diff
@@ -1,3 +1,68 @@
 import React, { useReducer, useContext } from 'react';
 import PropTypes from 'prop-types';
 import { createUseStyles } from 'react-jss';
+
+import UserContext from '../User/User';
+import { SaladContext } from '../SaladMaker/SaladMaker';
+
+// Items for a list of Salad options for the Salad Maker Interface
+const useStyles = createUseStyles({
```

Comment:
- The spacing for the useStyles object is all over the place and reduces the readability of the object. I would like to see tabs/spaces that are all in line for each object.
- Adding units for length and percentage values would increase readability and reduce confusion. It would also make it easier to match up styling across different components.